File: internal/sast/fixtures/python/log_positive.py
# internal/sast/fixtures/python/log_positive.py
import logging
from flask import Flask, request

app = Flask(__name__)
log = logging.getLogger(__name__)

@app.route("/login")
def login():
    user = request.args.get("user", "")
    log.info("Login attempt: " + user)
    return "ok"

File: internal/sast/fixtures/csharp/Log_positive.cs
// internal/sast/fixtures/csharp/Log_positive.cs
using Microsoft.AspNetCore.Mvc;
using Microsoft.Extensions.Logging;

public class LogController : Controller
{
    private readonly ILogger _log;
    public LogController(ILogger<LogController> log) { _log = log; }

    public IActionResult Login(string user)
    {
        _log.LogInformation("Login attempt: " + user);
        return Ok();
    }
}
